test/e2eprovider/server_test.go

```diff
@@ -1,99 +0,0 @@
-package e2eprovider
-
-import (
-	"context"
-	"encoding/json"
-	"reflect"
-	"testing"
-
-	"sigs.k8s.io/secrets-store-csi-driver/provider/v1alpha1"
-)
-
-func TestMount(t *testing.T) {
```

Comment:
It'll be good to have basic unit tests for the `e2e-provider`

Reply:
added tests